### ISSUE TITLE
No License file or license attribution comment on GitHub

### ISSUE THREAD

**Issue by rajsite** [jahredhope/promise-file-reader#9]
There is no License file included on GitHub or the NPM package. The npm package.json does seem to specify the ISC license. Is this the intended license for the project?

**Comment by rajsite** [comment]
A license attribution comment at the top of the `PromiseFileReader.js` similar to how jquery does it would be great: https://unpkg.com/jquery@3.3.1/dist/jquery.min.js

reproduced as follows
```
/*! jQuery v3.3.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

The `/*! some text */` pattern is recognized by a lot of minifiers as this comment needs to be preserved 😄

**Comment by jahredhope** [comment]
👍 
I believe ISC was the intended license. 
I'll look at adding the proper license documentation.

**Comment by jahredhope** [comment]
@rajsite I've added a LICENSE file.
I can see many other packages released under similar licenses that do not provide inline comments referring to the license. Is the linked change sufficient for your needs?

**Comment by rajsite** [comment]
That's great! Only thing left would be a published npm package with the license file added 👍 
(which also isn't a big rush since the License is listed in package.json)

**Comment by jahredhope** [comment]
Published in v1.0.2.
Thanks again for the help.
